WorkloadManagementSystem/DB/ElasticJobDB.py

```diff
@@ -78,23 +79,88 @@ def getJobParameters(self, jobID, paramList=None):
 
     return S_OK(resultDict)
 
-#############################################################################
-  def setJobParameter(self, jobID, key, value):
+############################################################################
+  def getJobParametersAndAttributes(self, jobID, paramList=None):
+    """ Get Job Parameters with Attributes defined for jobID.
+      Returns a dictionary with the Job Parameters.
+      If paramList is empty - all the parameters are returned.
+
+    :param self: self reference
+    :param int jobID: Job ID
+    :param list paramList: list of parameters to be returned
+    """
+
+    self.log.debug('JobDB.getParameters: Getting Parameters for job %s' % jobID)
+
+    jobParameters = ["JobID", "Name", "Value", "JobGroup", "Owner", "Proxy", "SubmissionTime", "RunningTime"]
+    resultDict = {}
+
+    if paramList:
+
+      query = {
+          "query": {
+              "bool": {
+                  "must": [
+                      {"match": {"JobID": jobID}}, {"match": {"Name": ','.join(paramList)}}]}},
+          "_source": jobParameters}
+
+    else:
+      query = {"query": {"match": {"JobID": jobID}}, "_source": jobParameters}
 
-    """ Set a parameter specified by name,value pair for the job JobID
+    gLogger.debug("Getting results for %d" % jobID)
+    result = self.query('jobelasticdb*', query)
+
+    if not result['OK']:
+      return result
+
+    sources = result['Value']['hits']['hits']
+    jobParameters.remove("JobID")
+
+    for source in sources:
+
+      jobID = source['_source']['JobID']
+      parametersDict = {}
+
+      for parameter in jobParameters:
+        parametersDict[parameter] = source['_source'][parameter]
+
+      resultDict[jobID] = parametersDict
+
+    return S_OK(resultDict)
+
+#############################################################################
+  def setJobParameter(self, jobID, key, value, **keywords):
```

Comment:
everywhere in dirac we use `**kwargs`, so I think it is better to stick to that notation